tools/harness-sniffer/README.md

```diff
@@ -0,0 +1,17 @@
+OpenThread Sniffer Integration with Thread Test Harness
+=============================
+After following the steps below, you will be able to run Test Harness with OpenThread sniffer.
+1) require python3 is installed on windows and in the system environment path.
+2) require TestHarness is updated on `ReportEngine.pyd`
+
+## Setup
+1. install pyspinel (for python3 only)
+    ```
+    git clone https://github.com/openthread/pyspinel.git
+    cd <path-to-pyspinel>
+    python setup.py install
```

Comment:
do we need to add '--extcap-path='  here?

Reply:
No. The scripts will be installed to Python installation path.

Reply:
ok